docker/Dockerfile

```diff
@@ -132,6 +132,7 @@ COPY ./pom.xml /root/app/pom.xml
 COPY ./api /root/app/api
 COPY ./yaci-indexer /root/app/yaci-indexer
 COPY ./test-data-generator /root/app/test-data-generator
+COPY ./.git /root/app/.git
```

Comment:
@AlksndrSlgb 

Error: buildx failed with: ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref 828adade-9126-45c5-a536-dc1f57fd1350::tqsecz9lbcvnpuxrbothetmi4: "/.git": not found

https://github.com/cardano-foundation/cardano-rosetta-java/actions/runs/9285469030/job/25550034552

Reply:
This pr should fix it